SeQuant/domain/mbpt/convention.cpp

```diff
@@ -105,6 +107,9 @@ void register_standard_instances() {
     // introduced in MPQC for GF, CT-F12, and other ad hoc uses
     IndexSpace::register_instance(declab(L"x"), IndexSpace::all_active, qnattr,
                                   do_not_throw);
+    // used in virtual projection CT-F12 theory methods in MPQC.
+    IndexSpace::register_instance(declab(L"c"), IndexSpace::complete_inactive_unoccupied, qnattr,
```

Comment:
too unsystematic ... should use greek letter at a minimum ... how about \gamma (related to g)